packages/components/color-picker/src/color-picker.tsx

```
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {forwardRef} from "@heroui/system";
+
+export interface ColorPickerProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)